### PR TITLE
[Raylife] LPS-148542 Raylife D2C Refactoring

### PR DESCRIPTION
**Pull request by victortrajano**

### CODE REVIEW

**Comment by victortrajano** [comment]
ci:test:sf

**Comment by victortrajano** [comment]
ci:test:relevant

**Comment by Mylena-Rodrigues** [comment]
Just starting reviewing :)

File: modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/hooks/useFormActionsDefault.js [hunk 1]
```diff
@@ -11,15 +11,17 @@
...
+import {useCallback, useContext, useEffect, useState} from 'react';
 import {useFormContext} from 'react-hook-form';
 import {STORAGE_KEYS, Storage} from '../../../common/services/liferay/storage';
 import {RAYLIFE_PAGES} from '../../../common/utils/constants';
 import {clearExitAlert} from '../../../common/utils/exitAlert';
 import {redirectTo} from '../../../common/utils/liferay';
 import {smoothScroll} from '../../../common/utils/scroll';
+import {AppContext} from '../context/AppContextProvider';
 import {createOrUpdateRaylifeApplication} from '../services/RaylifeApplication';
+
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/hooks/useFormActionsDefault.js:23]: Remove this white line

**Review by silasbsi — COMMENTED** [review]
LGTM :fire: But it needs a little adjustment

**Pull request closed without merging**